helm/templates/deployment.yaml

```diff
@@ -75,8 +75,9 @@ spec:
           livenessProbe:
             initialDelaySeconds: {{ int .Values.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ int .Values.livenessProbe.periodSeconds }}
-            tcpSocket:
```

Comment:
@ravisingal  (or anyone else) - do you have context on why the liveness probe was a tcp socket but the readiness probe was a health get?

Reply:
(he did not - one of those mysteries!)